src/pages/overview/pdf-services-api/howtos/webhook-notification.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Webhook Notification support for Adobe PDF Services APIs | Adobe PDF Services
+---
+# **Webhook Notification support for Adobe PDF Services APIs**
+
+Adobe PDF Services APIs have added support for webhook notification. After invoking the PDF Service API, the client need not required to continuously poll for the result of the job and they will get automatically notifies about job completion along with the status.
+
+The client needs to add an extra parameter in the input request called **notifier**. It is an array of webhook notification related information and each element contains following parameters:
+
+1. **type**: Type of notifier needs to be invoked. Currently supported notifier type is [CALLBACK](#callback-notifier).
+2. **data**: Information related to webhook notification data. It depends on type of notifier being used.
+
+## Sample Request Body
+```json
+{
+  "assetID": "{{assetID}}",
+  ....,
+  ///REST API Operation related details
+  ....
+  
+  "notifier": [
+    {
+      "type": "<NOTIFIER_TYPE>",
+      "data": {
+          //<NOTIFICATION_DATA>
+        }
+      }
+    }
+  ]
+}
+```
+
+<InlineAlert slots="text"/>
+<div>
+Webhook notification support is only available in REST APIs. It is currently <b>not</b> supported in SDKs.
+</div>
+
+## **CALLBACK Notifier**
```

Comment:
CALLBACK Notifier -> CALLBACK Webhook

Reply:
The callback webhook allows client to get the job completion notification on the https url. Once the job completes, we create a notification payload containing the job status and output asset details and invoke the HTTP POST call on the url provided by the client.